torch/_functorch/fx_minifier.py

```diff
@@ -95,13 +96,17 @@ def graph_fails(graph, inps):
     ConcreteProp(fail_f).propagate(*inps)
     if not graph_fails(failing_graph, inps):
         raise RuntimeError("Input graph did not fail the tester")
-    print(f"Started off with {cur_size} nodes")
+    print(f"Started off with {cur_size} nodes", file=sys.stderr)
 
     def _register_strategy(strategy: Callable, name: str):
         @wraps(strategy)
         def new_func(old_state: ReproState, granularity=1):
-            print()
-            print(f"Strategy: {name} (G: {granularity}) ({len(old_state.graph.nodes)} nodes, {len(old_state.inps)} inputs)")
+            print(file=sys.stderr)
```

Comment:
nit: If you're gonna jam all the prints towards `file=sys.stderr` consider doing something like

print_outstream = sys.stderr

And then if you change your mind, you only need to do flip flop it once

Reply:
it seems right for this to go to stderr though, it should always be line buffered and it's not intended to be parseable